EstateManagement.BusinessLogic/Manger/EstateManagementManager.cs

```diff
@@ -100,6 +110,49 @@ public class EstateManagementManager : IEstateManagementManager
             return merchantModel;
         }
 
+        /// <summary>
+        /// Gets the merchant balance.
+        /// </summary>
+        /// <param name="estateId">The estate identifier.</param>
+        /// <param name="merchantId">The merchant identifier.</param>
+        /// <param name="cancellationToken">The cancellation token.</param>
+        /// <returns></returns>
+        public async Task<MerchantBalance> GetMerchantBalance(Guid estateId,
+                                                              Guid merchantId,
+                                                              CancellationToken cancellationToken)
+        {
+            IEventStoreContext context = this.EventStoreContextManager.GetEventStoreContext(estateId.ToString());
+            
+            //var contextType = typeof(EventStoreContext);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/stuart_ferguson1/EstateManagement/pullRequest?prid=5660966)